Sources/OpenAPIRuntime/Conversion/Configuration.swift

```diff
@@ -96,6 +96,24 @@ extension JSONDecoder.DateDecodingStrategy {
     }
 }
 
+/// A type that allows custom content type encoding and decoding.
+public protocol CustomCoder: Sendable {
+
+    /// Encodes the given value and returns its custom encoded representation.
+    ///
+    /// - parameter value: The value to encode.
+    /// - returns: A new `Data` value containing the custom encoded data.
+    func customEncode<T: Encodable>(_ value: T) throws -> Data
```

Comment:
Please run the `./scripts/soundness.sh` script and check it doesn't produce any warnings, I think these comments aren't entirely consistent with the rest.

Reply:
Done